documentation/tutorial/src/triggerdump/triggerdump.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <ToolCommandName>triggerdump</ToolCommandName>
+    <RootNamespace>Diagnostics.TriggerDump</RootNamespace>
+    <Description>Trigger dump</Description>
+    <PackageTags>Diagnostic</PackageTags>
+    <PackageReleaseNotes>$(Description)</PackageReleaseNotes>
+    <TargetName>triggerdump</TargetName>
+	<OutputType>Exe</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="$(MSBuildThisFileDirectory)..\..\..\..\src\Tools\dotnet-trace\Extensions.cs" Link="Extensions.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(MSBuildThisFileDirectory)..\..\..\..\src\Microsoft.Diagnostic.Repl\Microsoft.Diagnostic.Repl.csproj" />
+    <ProjectReference Include="$(MSBuildThisFileDirectory)..\..\..\..\src\Microsoft.Diagnostics.Tools.RuntimeClient\Microsoft.Diagnostics.Tools.RuntimeClient.csproj" />
```

Comment:
And I'm not sure what good guidance is here. Eventually we plan to get this library on NuGet which would be a reasonable way for people to consume it, but we weren't planning on doing that until we've had a chance to refine the API, test it, and figure out what we want to support. I guess the best short-term option for people eager to get their hands dirty is fork the repo and build your own private copy.